draft-ietf-rtcweb-jsep.xml

```diff
@@ -2308,11 +2308,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                     subject to the following restriction: 
                     <list style="symbols">
 
-                        <t>The fields of the "o=" line MUST stay the
-                        same except for the &lt;session-version&gt;
-                        field, which MUST increment if the session
-                        description changes in any way, including the
-                        addition of ICE candidates.</t>
+                        <t>The fields of the "o=" line MUST stay the same except
+                        for the &lt;session-version&gt; field, which MUST
+                        increment to be greater than any other SDP offers in
+                        this session if the session description changes in any
+                        way, including the addition of ICE candidates. This
+                        means that if an Offer with version 12 was generated,
+                        then an Offer with version with 13 was generated, then
+                        the SDP was rolled back to the Offer with version 12,
+                        the next offer would have a version 14 or larger. </t>
```

Comment:
Can you rewrite this with more reference to the APIs? I.e., "If createOffer() returns a ..."